native_ref/java/pom.xml

```diff
@@ -8,6 +8,7 @@
         <groupId>com.github.fommil.netlib</groupId>
         <artifactId>native_ref</artifactId>
         <version>1.2-SNAPSHOT</version>
+        <relativePath>..</relativePath>
```

Comment:
I don't understand the point of these `<relativePath>` entries. I didn't need it when I built it... are you sure you read the documentation in the parent `pom.xml`?